core/src/banking_stage/packet_receiver.rs

```diff
@@ -73,20 +82,20 @@ impl PacketReceiver {
     }
 
     fn buffer_packets(
+        &self,
         ReceivePacketResults {
             deserialized_packets,
             new_tracer_stats_option,
             passed_sigverify_count,
             failed_sigverify_count,
         }: ReceivePacketResults,
-        id: u32,
         unprocessed_transaction_storage: &mut UnprocessedTransactionStorage,
         banking_stage_stats: &mut BankingStageStats,
         tracer_packet_stats: &mut TracerPacketStats,
         slot_metrics_tracker: &mut LeaderSlotMetricsTracker,
     ) {
         let packet_count = deserialized_packets.len();
-        debug!("@{:?} txs: {} id: {}", timestamp(), packet_count, id);
+        debug!("@{:?} txs: {} id: {}", timestamp(), packet_count, self.id);
```

Comment:
Was on the fence about removing `id` entirely since its' only used for this debug line.

Reply:
maybe, not sure if people use. the debug line to match with metrics by thread id.